### PR TITLE
Istio Service Mesh Article

### LINKED ISSUE #2495: [Resource] Article: Istio Service Mesh
Contributors and community members are encouraged to create articles under https://layer5.io/resources. A new article fitting the following description is requested to be written by a community member:

**URL**: /resources/service-mesh/what-is-istio-service-mesh
**Article Title:** Service Mesh: Istio
**Topic:** Create a page under /resources that generally describes Istio.
**Core points to convey in the article:** 
- Genesis
- Governance
- Architecture
- Special Features
- Excerpt of Functional, Non-Functional data from the service mesh landscape (/landscape)

_Articles under the /resources area are to be timeless and broadly helpful to visitors. As such, these articles carry no publish date or byline._

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by deepeshaburse**
Article on Istio Service Mesh. It includes:
- Genesis
- Governance
- Architecture
- Special Features

This PR fixes #2495 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 7de7498d178363e02e32dbb55720c3989278a379 at: https://63104370a49e952112ed495e--layer5.netlify.app

File: src/collections/resources/articles/istio-service-mesh/index.mdx [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+title: "Service Mesh: Istio"
+date: 2022-08-31 10:30:05 -0530
+author: Deepesha Burse
+thumbnail: ../../../../assets/images/service-mesh-icons/istio.svg
+darkthumbnail: ../../../../assets/images/service-mesh-icons/istio-white.svg
+description: Explanation of Istio
+type: Article
+published: false
```
Comment (debo19) [inline src/collections/resources/articles/istio-service-mesh/index.mdx:9]:
```suggestion
published: true
```

**Comment by l5io** [comment]
🚀 Preview for commit 884a4492e45daa9e7c0367e9446e79691884370c at: https://6315cf28e3237b4119c19d4d--layer5.netlify.app

**Comment by leecalcote** [comment]
Good stuff, @deepeshaburse. 👏

**Comment by deepeshaburse** [comment]
Thank you!

**Pull request merged**